### ISSUE TITLE
Edge case on if/unless modifier

### ISSUE THREAD

**Issue by fuadsaud** [rubocop/rubocop#264]
I have a chunk of code that goes like this:

``` ruby
if color.size == 3
  color = [(color[0] << 16) | (color[1] << 8) | color[2]]
end
```

Since this is a single line block, rubocop points that an if/unless modifier should be used.
Doing that would make the line get bigger than 79 characters, so you'd have to brake the expression. That's bad, because it's supposed to be a one liner, so I prefer to use the original layout. Another option would be to break that expression into more lines, but it wouldn't feel natural again. Maybe the if/unless modifier cop should check the resulting line length, but I'm not sure that would be good too.
